.github/workflows/poetry.yml

```diff
@@ -91,13 +92,16 @@ jobs:
           - "3.10" # highest supported
           - "3.9"
           - "3.8"
-          - "3.7"
-          - "3.6" # lowest supported
+          - "3.7" # lowest supported
         toxenv-factor: ['locked']
         include:
+          - # test with py37 ubuntu20
+            os: ubuntu-20.04
+            python-version: '3.7'
+            toxenv-factor: 'locked'
           - # test with the lowest dependencies
-            os: 'ubuntu-latest'
-            python-version: '3.6'
+            os: ubuntu-20.04
```

Comment:
why 20.04? is py37 not working in `latest` anymore

Reply:
Uh, I think I've cherry-picked this wrong from the previous PR. Set to `latest` now.